packages/binding-netconf/src/async-node-netconf.ts

```diff
@@ -37,15 +37,15 @@ export class Client {
         this.router = null;
     }
 
-    getRouter() {
+    getRouter(): any {
```

Comment:
isn't this a `nodeNetconf.Client` ?

Reply:
Hmm, I think the return value of this getter is only the parameters needed to *instantiate* a `nodeNetconf.Client`. Changing line 34 to 

```
private router: nodeNetconf.Client;
```

leads to a number of errors, as this `router` variable is actually passed to a `nodeNetconf.Client` constructor in the `openRouter()` method. Maybe the `router` variable should be renamed to something like `routerParameters` instead and typed with an appropriate interface.